modules/nested-elements/module.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Elementor\Modules\NestedElements;
+
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends \Elementor\Core\Base\Module {
+
+	public static function get_experimental_data() {
+		return [
+			'name' => 'nested-elements',
+			'title' => esc_html__( 'Nested Elements', 'elementor' ),
+			'description' => esc_html__( 'Adds a new Elementor nested elements that allows creating nested tabs, nested according , etc...', 'elementor' ),
+			'release_status' => Experiments_Manager::RELEASE_STATUS_ALPHA,
+			'default' => Experiments_Manager::STATE_INACTIVE,
+			'dependency' => [
+				'container',
```

Comment:
```suggestion
```
Fix dependency, now uses full class name.

Reply:
@matipojo, I made dependencies based on their module class, now see that container does not have module, and experiment registered via the manager itself, can we add module in core for the container, or should we make experiments manager work with feature name instead?